homeassistant/components/hanna/quality_scale.yaml

```diff
@@ -0,0 +1,73 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration doesn't add actions.
+  appropriate-polling:
+    status: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  entity-unique-id:
+    status: done
+    comment: |
+      This integration uses the device identifier as unique ID. The device identifier is unique for each Hanna device.
+  has-entity-name:
+    status: done
+    comment: |
+      Entities use has_entity_name = True. All Sensors have a name.
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: todo
+  docs-configuration-parameters: todo
+  docs-installation-parameters: todo
+  entity-unavailable: todo
+  integration-owner: todo
+  log-when-unavailable: todo
+  parallel-updates: todo
+  reauthentication-flow: todo
+  test-coverage: todo
+
+  # Gold
+  devices: todo
+  diagnostics: todo
+  discovery-update-info: todo
+  discovery: todo
+  docs-data-update: todo
+  docs-examples: todo
+  docs-known-limitations: todo
+  docs-supported-devices: todo
+  docs-supported-functions: todo
+  docs-troubleshooting: todo
+  docs-use-cases: todo
+  dynamic-devices: todo
+  entity-category: todo
+  entity-device-class: todo
+  entity-disabled-by-default: todo
+  entity-translations: todo
+  exception-translations: todo
+  icon-translations: todo
+  reconfiguration-flow: todo
+  repair-issues: todo
+  stale-devices: todo
+
+  # Platinum
+  async-dependency: todo
+  inject-websession: todo
+  strict-typing: todo
```

Comment:
I think some of these rules are already fulfilled in this initial integration.
Could you please take a pass over the rules and update accordingly?
Those who are not yet fulfilled can be left as "todo".

Reply:
Updated :)
Saw that I was calling `await coordinator.async_shutdown()` in `async def async_unload_entry` but the `async_shutdown()` was never defined in the coordinator. Added a simple del on the api_client.
will implement a .close() method in a future release of the hanna_cloud package.